### ISSUE TITLE
Add custom location of docker containers

### ISSUE THREAD

**Issue by a-komarev** [jenssegers/captain#8]
Now it's searching for docker containers in `$HOME` directory. It will be good to have an option to add additional directories where to search for them or ability to define location of containers one by one.

`captain add /mnt/something/interesting/and/useful`

**Comment by jenssegers** [comment]
Current idea is a `~/.captain` file. But I like your idea about being able to add specific folders.

Maybe something like:

`captain ships add ~/Projects`

**Comment by a-komarev** [comment]
Yeah, I've thought about `~/.captain` file too. But both of this ways could work together. `add` command could append directory to the `~/.captain` file.

**Comment by tom472** [comment]
Project is very cool and useful - Thanks for making it.
This idea was the first thing I thought of when trying it out. Great to see it has already been thought of and might be added as an option.

**Comment by Grummfy** [comment]
see also #3

**Comment by ta-tikoma** [comment]
I liked this application. But I have this path: 
`/home/my-name/work/company-name/{project-name}/docker/local/docker-compose.yml`
I wanna say `captain start project-name` . 
May be add command `captain add project-name path-to-docker-compose-file` ?
